lib/SPIRV/libSPIRV/SPIRVValue.h

```diff
@@ -175,40 +175,59 @@ template <spv::Op OC> class SPIRVConstantBase : public SPIRVValue {
   float getFloatValue() const { return Union.FloatVal; }
   double getDoubleValue() const { return Union.DoubleVal; }
   unsigned getNumWords() const { return NumWords; }
-  SPIRVWord *getSPIRVWords() { return Union.Words; }
+  const std::vector<SPIRVWord> &getBigValWords() { return BigValWords; }
 
 protected:
+  constexpr static SPIRVWord FixedWC = 3;
+  constexpr static unsigned WordBitWidth = sizeof(SPIRVWord) * CHAR_BIT;
```

Comment:
I guess we already have this constant somewhere in this repo.

Reply:
I didn't find it. I created a new one. Please let me know if I missed it somewhere.